pages/api/auth/[...nextauth].js

```diff
@@ -21,13 +20,27 @@ const authOptions = {
           return context
         }
       },
-      profile({ provider, tokens }) {
+      profile({ tokens }) {
         return tokens.user
       }
     }
   ],
   theme: {
     logo: '/Logo.svg', // requires a file in the public folder
+  },
+  callbacks: {
+    async jwt({ token, account, user }) {
+      // Triggered on the initial sign
+      if (account && user) {
+        token.accessToken = account.access_token
```

Comment:
setting the access token on the `token` object here, makes it available on the `token` argument in the `session` function below.